Interview/demo.md

```diff
@@ -63,6 +63,7 @@ The following requirements should be met:
 
 * Use Swift 4.0 or above.
 * The information (posts and post details) should be available offline. It's assumed that, if it's the first time you are accessing the app and you are offline, you shouldn't see any data.
+* **Have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).**
```

Comment:
```suggestion
* **Have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).**
* **No retain cycle.**
```

Reply:
Can we just make it plural? 😛 `No retain cycles.`